### DISCUSSION TITLE
如何使用阿里云通义千问(qwen)

### DISCUSSION THREAD

**Discussion by lNwNl** [ChatGPTNextWeb/NextChat#5038] [Q&A]
看起来在最新(v2.13.0)版本的release中，已经加入了对于阿里云模型api的支持，但是我不确定正确的使用方法应该是什么，具体来说：
1. 接口地址应该填入什么呢？（或保持默认？）
2. 接口密钥是否是从[阿里云百炼](https://bailian.console.aliyun.com/?apiKey=1#/api-key)创建？
3. 是否支持qwen-long？qwen-long支持很长的上下文，并能够解析上传的文档
十分感谢大家的帮助！

**Comment by yrei1988** [comment]
1，地址用默认
2，apikey就是到阿里百炼开通

**Reply by lNwNl** [reply]
感谢回复，但是对于使用release安装的方式，这样仍旧无法正常使用 #5047

**Reply by KevinCai76** [reply]
1，地址用默认: 是https://dashscope.aliyuncs.com/api/ 这个吗？不行，报"Failed to fetch"

**Comment by yrei1988** [comment]
发自我的iPhone


------------------ 原始邮件 ------------------
发件人: KevinCai ***@***.***&gt;
发送时间: 2024年8月28日 11:59
收件人: ChatGPTNextWeb/ChatGPT-Next-Web ***@***.***&gt;
抄送: yrei1988 ***@***.***&gt;, Comment ***@***.***&gt;
主题: Re: [ChatGPTNextWeb/ChatGPT-Next-Web] 如何使用阿里云通义千问(qwen) (Discussion #5038)





 
1，地址用默认: 是https://dashscope.aliyuncs.com/api/ 这个吗？不行，报"Failed to fetch"
